src/averageAges.js

```diff
@@ -55,7 +69,19 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const children = onlyWithSon
+    ? people.filter(p => p.sex === 'm'
+      && people.some(m => m.name === p.mother))
+    : people.filter(p => people.some(m => m.name
+      === p.mother));
```

Comment:
Unnecessary line break. 
const children = onlyWithSon
    ? people.filter(p => p.sex === 'm' && people.some(m => m.name === p.mother))
    : people.filter(p => people.some(m => m.name === p.mother));
Pay attention, you have different line break in all 3 functions. Stick to one structure.